site/gatsby-site/src/components/ui/MdxComponents.js

```diff
@@ -3,8 +3,16 @@ import isString from 'lodash/isString';
 import Link from './Link';
 import SensitiveImage from '../images/sensitive';
 
-const slug = (title) =>
-  isString(title) ? title.toLowerCase().replace(/\s+/g, '') : title.props.children;
+const slug = (title) => {
+  console.log(`title`, title);
+  if (isString(title)) {
+    return title.toLowerCase().replace(/\s+/g, '');
+  }
+  if (Array.isArray(title)) {
+    return title.map((e) => slug(e)).join('');
+  }
```

Comment:
In which case do we have this title as an Array?

Reply:
When a heading contains a link, the text and the link become array elements. E.g.

```markdown
### Report 1: [Knightscope Issues Field Incident Report](/cite/51#r1765)
```

Reply:
Got it!